domain.yml

```diff
@@ -15,63 +37,95 @@ entities:
 
 responses:
   utter_greet:
-  - text: "Hi! It's time for your daily wellness check. Tracking healthy habits is a great way to measure your progress over time. Would you like to answer a few questions about your health?"
+  - text: Hi! It's time for your daily wellness check. Tracking healthy habits is a great way to measure your progress over time. Would you like to answer few questions about your health?
+
+  utter_cheer_up:
+  - text: 'Here is something to cheer you up:'
+    image: https://i.imgur.com/nGF1K8f.jpg
+
+  utter_did_that_help:
+  - text: Did that help you?
+
+  utter_happy:
+  - text: Great, carry on!
 
   utter_goodbye:
-  - text: "See you tomorrow. Bye!"
+  - text: Bye
 
   utter_iamabot:
-  - text: "I am a bot, powered by Rasa."
-  
-  utter_ask_confirm_exercise:
+  - text: I am just another bot trying to help you.
+
+  utter_ask_health_form_confirm_exercise:
   - text: Did you exercise yesterday? Don't sweat it if you didn't run a marathon - walks count!
-  utter_ask_exercise:
-  - text: What kind of exercise did you do 💪 ?
-  utter_ask_sleep:
-  - text: "How much sleep did you get 💤  last night?"
-  utter_ask_diet:
-  - text: "Did you stick to a healthy diet 🥦  yesterday?"
-  utter_ask_stress:
-  - text: "What is your stress level right now 🧘 ?"
-    buttons:
-      - title: "low"
-        payload: '/inform{"stress": "low"}'
-      - title: "medium"
-        payload: '/inform{"stress": "medium"}'
-      - title: "high"
-        payload: '/inform{"stress": "high"}'
-  utter_ask_goal:
-  - text: "Setting goals - even small ones - is a great way to focus your day. What do you want to accomplish today 🥇 ?"
-  utter_slots_values:
-  - text: "You've submitted the following answers:\n
+
+  utter_ask_health_form_sleep:
+  - text: How much sleep did you get 💤 last night?
+
+  utter_ask_health_form_exercise :
+  - text: What kind of exercise did you do?
+
+  utter_ask_health_form_diet:
+  - text: Did you stick to a healthy diet 🥦 yesterday?
+
+  utter_ask_health_form_stress:
+  - text: Is your stress level low, medium or high 🧘🏼?
+
+  utter_ask_health_form_goal:
+  - text: Setting goals - even small ones - is a great way to focus your day. What do you want to accomplish today 🥇 ?
+
+  utter_slots_value:
+  - text: "Here's your daily wellness log:\n
             - Exercised?: {confirm_exercise}\n
             - Type of exercise: {exercise}\n
             - Sleep: {sleep}\n
             - Stuck to a healthy diet?: {diet}\n
             - Stress level: {stress}\n
             - Goal: {goal}"
+
   utter_no_worries:
-  - text: "No problem :)"
+  - text: No Problem :)
+
   utter_ask_continue:
-  - text: "Sorry, I don't quite understand. Do you want to continue?"
+  - text: Sorry, I don't quire understand. Do you want to continue?
```

Comment:
Looks like there's a small typo here :)

Reply:
Nice catch, so silly of me, fixing it  :)